grid-client/workloads/network.go

```diff
@@ -24,6 +24,13 @@ type UserAccess struct {
 	PublicNodeEndpoint string
```

Comment:
`NewNetworkFromWorkload` also needs to be updated to set metadata stored in the workload to znet struct, right?

Reply:
Sure it should. I didn't add it because i think the znet retrieval in the grid client right now is somehow redundant. It retrieves information from only one workload, which is not entirely right i think.
Will update this, it would be really helpful.
Thanks

Reply:
I think it can be parsed from a zos workload so the functions should do it